src/models/linear_network.py

```diff
@@ -144,6 +155,24 @@ def predict(self) -> Tuple[Dict[str, Dict[str, np.ndarray]], Dict[str, np.ndarra
 
         return test_arrays_dict, preds_dict
 
+    def _get_background(self, sample_size: int = 100) -> torch.Tensor:
+
+        print('Extracting a sample of the training data')
+
+        train_dataloader = DataLoader(data_path=self.data_path,
```

Comment:
this background is randomly generated right? Was one of your comments about SHAP values that the background can be especially selected if there is a particularly explainable background to compare results to?

Reply:
Yes; we could add a defined background in the future for sure.